docs/jitserver_tuning.md

```diff
@@ -42,14 +42,16 @@ The client-session caches are deleted when the clients terminate, but this can h
 
  To enable this feature, specify the [`-XX:+JITServerUseAOTCache`](xxjitserveruseaotcache.md) command line option, both at the server and at the client JVM.
 
- A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). If the client doesn't specify a name for the AOT cache, the server uses a cache named `default`.
+ A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command-line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). If the client doesn't specify a name for the AOT cache, the server uses a cache named `default`.
 
  The maximum amount of memory that all the AOT cache instances combined can use at the server is 300 MB, by default. You can change this value by using the [`-XX:JITServerAOTmx=<size>`](xxjitserveraotmx.md) option. When the cache size reaches the specified limit, new clients cannot create new AOT cache instances or add new compiled methods to the existing AOT cache instances.
 
- Current limitations:
+ JITServer AOT cache can be reused by different instances of its JITServer server by enabling the AOT persistence feature with the [`-XX:+JITServerAOTCachePersistence`](xxjitserveraotcachepersistence.md) option. With this feature enabled, the JITServer server periodically saves its AOT caches to files. The server saves a new file when the number of extra AOT methods added to the in-memory cache since the last save equals or exceeds the number set in the `-Xjit:aotCachePersistenceMinDeltaMethods=<number of methods>` option. The default number is 200 methods.
+
+ If the specified number of extra methods gets added to the in-memory cache fast (for example, during the ramp-up phase of an application), it results in multiple save operations within a short period. To control the frequency of the save operations, you can specify the minimum time between two consecutive JITServer AOT cache save operations in the `-Xjit:aotCachePersistenceMinPeriodMs=<milliseconds>` option. The server saves a new file only if the time exceeds the set time. The default minimum time is 10000 milliseconds.
```

Comment:
This sentence seems clunky: "The server saves a new file only if the time exceeds the set time"
Maybe we should remove it because it does not bring any additional information.